src/App.js

```diff
@@ -1,10 +1,23 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
 import './App.css';
+import Navbar from './components/Navbar/Navbar';
 
 function App() {
   return (
-    <div className="App">
-      <h1 className="text-lg text-orange-700 font-semibold">Hello world</h1>
-    </div>
+    <>
+      <BrowserRouter>
+        <Navbar />
+        <div className="container">
```

Comment:
- As shown in the image below the container element is taking too much width. And scrolling the horizontal scrollbar at the bottom to the right direction will make the container content intersect with the navbar element. 
![fix2](https://user-images.githubusercontent.com/88746054/229340003-a75a03af-10ca-446b-aad4-0fe1e0c88243.png)

- I assume this problem happened on my pc because I have a small screen size.
- deleting `position: fixed;` from `.navbar` class and adding `display: flex` to `#root` fixed this issue for me.

Reply:
Yes, Thank you for the suggestion. Done :+1: